packages/react/src/FormErrorList/FormErrorList.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+import { Alert } from '../Alert'
+import { HeadingLevel } from '../Heading/Heading'
+import { LinkList } from '../LinkList'
+
+export type FormErrorListProps = {
+  errors: { id: string; label: string }[]
```

Comment:
This is very basic. Do we know how developers want to pass errors to a component like this?

Reply:
This component should only render a list of links, so it shouldn't be more complicated than this, imo. Or do you expect that more functionality is required?

Reply:
There might be, for example there could be an explanation in the form of a paragraph besides a link.